app/src/main/res/layout/fragment_subscriptions.xml

```diff
@@ -10,32 +11,81 @@
         android:id="@+id/toolbar"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:layout_alignParentTop="true"
         android:minHeight="?attr/actionBarSize"
         android:theme="?attr/actionBarTheme"
-        android:layout_alignParentTop="true"
-        app:title="@string/subscriptions_label"
-        app:navigationIcon="?homeAsUpIndicator" />
+        app:navigationIcon="?homeAsUpIndicator"
+        app:title="@string/subscriptions_label" />
 
-    <TextView
-        android:id="@+id/feeds_filtered_message"
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
We don't use a constraintlayout anywhere else. Do you do anything that cannot be done with RelativeLayout here?

Reply:
I was hoping you would know. I tried with a RelativeLayout, couldn't get it to work. I could try again.